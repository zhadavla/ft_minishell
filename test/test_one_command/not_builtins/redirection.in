ls | grep minishell > outfile | cat outfile 