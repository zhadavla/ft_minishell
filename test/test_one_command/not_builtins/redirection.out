minishell$ ls | grep minishell > outfile | cat outfile 
minishell$ exit
